apps/omg/test/omg/state_test.exs

```diff
@@ -27,7 +27,7 @@ defmodule OMG.StateTest do
 
   require Utxo
 
-  @eth OMG.Eth.zero_address()
+  @eth <<0::160>>
```

Comment:
why don't you want to use a common type?

Reply:
I don't think it adds anything but it increases coupledness with modules and decreases readability, I like tests self contained.